Model/EventsForThirdPartyModules.php

```diff
@@ -758,8 +758,27 @@ class EventsForThirdPartyModules
                     "sendClasses" => ["Mirasvit\Rewards\Model\Config"],
                     "boltClass" => Mirasvit_Rewards::class,
                 ],
+                [
```

Comment:
```suggestion
                'MW_RewardPoints' => [
```
for easier navigation